src/aggregator/aggregator/handler/config.go

```diff
@@ -178,15 +180,17 @@ func (c *dynamicBackendConfiguration) newProtobufHandler(
 	for _, filter := range c.Filters {
 		sid, f := filter.NewConsumerServiceFilter()
 		p.RegisterFilter(sid, f)
-		logger.Infof("registered filter for consumer service: %s", sid.String())
+		logger.Info("registered filter for consumer service", zap.Stringer("service", sid))
```

Comment:
Make these strings consts?

Reply:
I think that makes the code harder to read in this case. Like you want to see the full log line in the source code when you're debugging. No point to moving the constant away from the line

Reply:
fair enough